components/Div/index.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { CodeTabs } from '../CodeTabs';
+
+const Div = props => {
+  if (props.className === 'code-tabs') {
+    return <CodeTabs {...props} />;
+  }
```

Comment:
Not a huge fan of baking this code tabs-specific markup in to the `Div` component… 🤔

Reply:
Oops, just realized I checked in a bunch of update snapshots. The custom `Div` was breaking things...